src/parser/monk/mistweaver/modules/spells/InvokeChiJi.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+
+import Analyzer, { SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import Combatants from 'parser/shared/modules/Combatants';
+
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+
+import { formatNumber } from 'common/format';
+import { TooltipElement } from 'common/Tooltip';
+
+const debug = false;
+
+class InvokeChiJi extends Analyzer {
+  static dependencies = {
+    conbatants: Combatants
+  };
+
+  petID = null;
+  healing = 0;
+  overhealing = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER).spell(SPELLS.GUST_OF_MISTS_CHIJI), this.handleGust);
+    debug && this.addEventListener(Events.fightend, this.fightEndDebug);
+  }
+
+  handleGust(event) {
+    this.healing += (event.amount || 0) + (event.absorbed || 0);
+  }
+
+  fightEndDebug() {
+    console.log('Chi-Ji summoned!');
+  }
+
+  statistic() {
+    if (this.selectedCombatant.hasTalent(SPELLS.INVOKE_CHIJI_THE_RED_CRANE_TALENT.id)) {
```

Comment:
move this check up to the constructor:

```javascript
this.active = this.selectedCombatant.hasTalent(SPELLS.INVOKE_CHIJI_THE_RED_CRANE_TALENT.id);
```
and put it before the addEventListeners 👍 

This will automatically not add a statistic if the module is not active, and wont bind the listeners